Resources/ServerInfo/Guidebook/ServerRules/SpaceLaw/SpaceLaw.xml

```diff
@@ -21,14 +21,14 @@
   A departmental search is the sweep of an entire area or department for contraband. It is recommended that the officers be extremely thorough, checking all lockers, crates, and doors. These can only be done with permission or, ideally, a warrant signed by the department head or highest-ranking command staff, which is the captain in most cases.
 
   ## Implantation
-  Any prisoner in custody can be subjected to implantation or implant removal procedures, so long as it's within reason. The process of adding an implant should not prolong the detainees sentence, meaning you can not hold them longer to administer the implant, unless stated otherwise. A former inmate can be requested to undergo implantation at a later point in time if they fit the circumstances during their confinement, they must comply. The following have been listed out with special circumstances, anything not in this list can still be applied, given proper legal context. A prisoner can still receive implantation procedures without meeting the circumstances if they give their clear permission.
+  Any prisoner in custody can be subjected to implantation or implant extraction procedures, so long as it's within reason. The process of adding or extracting an implant should not prolong the detainees sentence, meaning you can not hold them longer to perform the procedure, unless stated otherwise. A former inmate can be requested to undergo implantation/extraction at a later point in time; if they fit the circumstances during their confinement, they must comply. The following have been listed out with special circumstances, anything not in this list can still be applied, given proper legal context. A prisoner can still receive implantation/extraction procedures without meeting the circumstances if they give their clear permission.
```

Comment:
```suggestion
  Any prisoner in custody can have an implant implanted, so long as it's within reason. Unless stated otherwise, the process of implanting an implant must not prolong the detainee's sentence. A  detainee must give clear permission to be implanted, except during the special circumstances below. At any point after release, an officer may ask a former detainee to undergo implantation. They must comply if they would have fulfilled the circumstances below during their confinement. 
```